cloudify_cli/tests/resources/blueprints/bad_blueprint/blueprint_catalog.yaml

```diff
@@ -5,9 +5,5 @@ description: >
     with a local `cfy blueprints validate`
 
 imports:
-  - http://www.getcloudify.org/spec/cloudify/6.3.0.dev1/types.yaml
```

Comment:
Not calling out to the internet is such a massive win for this it wants merging yesterday.